code/modules/martial_arts/martial.dm

```diff
@@ -428,5 +446,31 @@
 		return ..()
 	return 0
 
+/obj/screen/combo
+	icon_state = ""
+	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
+	screen_loc = ui_combo
+	layer = ABOVE_HUD_LAYER
+	var/timerid
+
+/obj/screen/combo/proc/clear_streak()
+	cut_overlays()
+	icon_state = ""
+
+/obj/screen/combo/update_icon_state(streak = "")
+	clear_streak()
```

Comment:
😡 overlays will be cut automatically at this stage if you properly split out overlay operations to `UPDATE_OVERLAYS`, due to how `update_icon` works when it calls `UPDATE_OVERLAYS`. If you pull overlay updates into another proc, you can just set `icon_state` to nothin and carry on without needing to call a whole other proc here.

Reply:
I forgor we did this